### PR TITLE
Update sustainability labels evaluations

### PR DESCRIPTION
**Pull request by AdriaSG**
I had added the new sustainability-labels-evaluation file as csv, also sustainability-labels.json was updated. Code inside sustanability_labels/__init__.py had been edited to read new file and merge data. Finally, the table structure was updated in sustanability_labels/bootstrap_database.py; as well as in core/domain.py and database.tables.py.

### CODE REVIEW

**Review by se-jaeger — CHANGES_REQUESTED** [review]
Looks good! However, I pushed a change to the csv file. The column names have `:` instead of `_` could you change this? Thanks!

File: core/core/domain.py [hunk 1]
```diff
@@ -66,9 +66,20 @@ class SustainabilityLabel(BaseModel):
...
+    eco_inputs: Optional[conint(ge=0, le=100)]  # type: ignore
+    eco_quality: Optional[conint(ge=0, le=100)]  # type: ignore
+    eco_energy: Optional[conint(ge=0, le=100)]  # type: ignore
+    eco_waste_air: Optional[conint(ge=0, le=100)]  # type: ignore
+    eco_environmental_management: Optional[conint(ge=0, le=100)]  # type: ignore
+    social_labour_rights: Optional[conint(ge=0, le=100)]  # type: ignore
+    social_business_practice: Optional[conint(ge=0, le=100)]  # type: ignore
+    social_social_rights: Optional[conint(ge=0, le=100)]  # type: ignore
+    social_company_responsibility: Optional[conint(ge=0, le=100)]  # type: ignore
+    social_conflict_minerals: Optional[conint(ge=0, le=100)]  # type: ignore
```
[the remark this answers is not in the document]
Comment (se-jaeger) [reply core/core/domain.py:82]: Hopefully using `alias` allows to export the proper format, which is important to take care of.

[the remark this answers is not in the document]
Comment (AdriaSG) [reply core/core/domain.py:82]: What do you think about just renaming the original labels to not use ":" but "_" as I just did in sustanability_labels/init.py

[the remark this answers is not in the document]
Comment (se-jaeger) [reply core/core/domain.py:82]: agreed 👍🏼

**Review by se-jaeger — CHANGES_REQUESTED** [review]
Changing the labels and updating the timestamp https://github.com/calgo-lab/green-db/blob/e4a9732cf3a6b8034ac7f955878f25eee284b6e1/core/core/sustainability_labels/bootstrap_database.py#L39 does not update the table.

This is probably because of this if-statement https://github.com/calgo-lab/green-db/blob/e4a9732cf3a6b8034ac7f955878f25eee284b6e1/database/database/connection.py#L203-L207
Could you take care of this? - Maybe we need to change the primary key to be a tuple `(id, timestamp)`?

I don't mind if add all labels with the new timestamp even if we change the evaluation of a single label.

File: core/core/sustainability_labels/sustainability-labels-evaluation.csv [hunk 2]
```diff
@@ -0,0 +1,79 @@
+id,name,category,cred:credibility,eco:chemicals,eco:lifetime,eco:water,eco:inputs,eco:quality,eco:energy,eco:waste_air,eco:environmental_management,social:labour_rights,social:business_practice,social:social_rights,social:company_responsibility,social:conflict_minerals
```
[the remark this answers is not in the document]
Comment (AdriaSG) [reply core/core/sustainability_labels/sustainability-labels-evaluation.csv:1]: Thanks for updating it, I did manual changes on mine to have names with _ and forgot to change it back to the original.

**Comment by se-jaeger** [comment]
Oha and forgot to mention that  the column `eco_inputs` is empty for all rows which is weird because some rows have evaluations for it.

**Comment by se-jaeger** [comment]
And please update the Readme accordingly to the changes of the sustainability labels table.

**Comment by AdriaSG** [comment]
I made changes requested, what do you think about the updates on green-db/database/database/connection.py?

File: database/database/connection.py [hunk 3]
```diff
@@ -197,16 +197,30 @@ def __init__(self) -> None:
...
+                    )
+                    .filter(
+                        SustainabilityLabelsTable.id == label.id,
+                        SustainabilityLabelsTable.timestamp == label.timestamp,
+                    )
+                    .first()
+                ):
+                    db_session.query(SustainabilityLabelsTable).filter(
+                        SustainabilityLabelsTable.id == label.id
+                    ).update(label.dict())
```
Comment (se-jaeger) [inline database/database/connection.py:216]: I would prefer it if we do not update existing rows. Creating new ones allows us to version the labels' evaluations.

File: database/database/connection.py [hunk 4]
```diff
@@ -197,16 +197,30 @@ def __init__(self) -> None:
...
+
             for label in sustainability_labels:
-                if (  # If label does not exist
+                if (  # If label exist with different timestamp, row will be updated
+                    not db_session.query(
+                        SustainabilityLabelsTable.id, SustainabilityLabelsTable.timestamp
+                    )
+                    .filter(
+                        SustainabilityLabelsTable.id == label.id,
+                        SustainabilityLabelsTable.timestamp == label.timestamp,
```
Comment (se-jaeger) [inline database/database/connection.py:210]: Here, it's only necessary to query for the `timestamp`.

**Review by se-jaeger — CHANGES_REQUESTED** [review]
I added some inline comments :)

I guess, if we do not update rows and create new ones instead, it is necessary to change the primary to the tuple I mentioned.

File: database/database/connection.py (hunk 4 above)
Comment (AdriaSG) [reply database/database/connection.py:210]: File changed in [5367245](https://github.com/calgo-lab/green-db/pull/73/commits/536724540c938edbfbc7ba78342ac315edfe1813)

File: database/database/connection.py (hunk 3 above)
Comment (AdriaSG) [reply database/database/connection.py]: File changed in [5367245](https://github.com/calgo-lab/green-db/pull/73/commits/536724540c938edbfbc7ba78342ac315edfe1813)

**Review by se-jaeger — APPROVED** [review]
Added a minor change. If the timestamp changes, we add all the labels again (even if there are no changes). This makes it easier to get all the latest label versions similar to the products. Hope this is ok for you?

**Pull request merged**